ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+
+![Figure: Ansible and CVP](../_media/getting-started/Ansible-and-CVP-httpapi.png)
+
+## What is Ansible AVD (Ansible collection for Arista Validated Designs)
+While Ansible is the automation engine, Ansible AVD is a “collection” added to Ansible.
+It's an open source project hosted on GitHub and maintained by Arista, letting users contribute directly. 
+A “collection” is a distribution format for Ansible content that can include roles, modules, and plugins.
+These topics are most likely new to you, but do not worry, they will be described in later sections.
+For now, just remember that a “collection” is a way to easily share and distribute Ansible data.
+You can install and use collections through Ansible Galaxy, which is Ansible’s official hub for sharing Ansible content.
+
+![Figure: Arista Leaf Spine topology](../_media/getting-started/Ansible-AVD.png)
+
+In the illustration above, the Ansible AVD collection is shown as the box with the red background on the right hand side.
+
+When designing an EVPN Layer 3 Leaf Spine (L3LS) fabric you don't need to be an expert on Arista Design and Deployment Guides.
+All those rules and guidelines are integrated in the Ansible collection for Arista Validated Designs (AVD), or in short: Ansible AVD.
+
+Ansible AVD is an opinionated collection, meaning that field experiences and best practices from actual deployments at Arista customers are included.
+With Ansible AVD, users get up and running quickly without having to invent every part of the network configuration from scratch.
+This means only the most basic information must be provided by the user to get a new fabric up and running.
+ 
+Parameters which need to be defined by the user include:
+- Username/password for network devices and Cloudvision
+- OOB management IP range
+- IP scope for link addresses
+- IP scope for loopback interfaces
+- VLAN and VNI ranges
+- Device hostnames
+- Which devices are acting in each role, such as spine or leaf.
+
+All are details that must be decided upon in any fabric design. 
+
+In a standard spine-leaf topology, Ansible AVD takes care of all the work generating the full EOS configurations past this high level description of names and numbering for the network.
+
+## When to use Ansible AVD and when not to
+It’s important to mention when to, and perhaps more importantly, when not to use Ansible AVD.
+
+Automating provisioning of a network infrastructure makes most sense when the network is designed and built using clearly defined,
+repeatable patterns. It makes less sense when a network infrastructure is using a non-uniform design.
+Although you can use Ansible AVD to provision a non-uniform network while harvesting some benefits,
+you’re creating a lot of manual work and customization of Ansible AVD to achieve what you need,
+which may ultimately be more work than manually managing the network when every device is its own special case.
+
+## Change your mindset and culture
```

Comment:
I think this point could be more generic in terms of mindset and culture and not to a specific tool. Consider removing mentions of specific tools (Ansible) from this section and include terms like NetDevOps, Infrastructure-as-code (IaC) and CI/CD. You can find some inspiration here [What is NetDevOps](https://www.packetcoders.io/what-is-netdevops/)

Reply:
It is a good idea to give a taste of the "current state of the art" of NetDevOps, but we need to be careful as this can be a very long section. I suggest we add some concepts, specially the ones we see more traction on field, but also keep the section as it is now because it gives a good overview of the way we are expecting to use AVD (How AVD works best).